llama_stack/distribution/routers/routing_tables.py

```diff
@@ -465,6 +465,26 @@ async def register_eval_task(
         )
         await self.register_object(eval_task)
 
+    async def DEPRECATED_list_eval_tasks(self) -> ListEvalTasksResponse:
+        raise DeprecationWarning("Use /eval/tasks instead")
+
+    async def DEPRECATED_get_eval_task(
+        self,
+        eval_task_id: str,
+    ) -> Optional[EvalTask]:
+        raise DeprecationWarning("Use /eval/tasks instead")
```

Comment:
Are we re-directing old endpoints to new ones somewhere?

Reply:
We now deprecate `/eval-tasks` endpoints to `/eval/tasks`